### PR TITLE
PBM-1265: Fix pitr-physical

### PR DESCRIPTION
**Pull request by egegunes**
[PBM-1265](https://jira.percona.com/browse/PBM-1265) [Powered by Pull Request Badge](https://pullrequestbadge.com/?utm_medium=github&utm_source=percona&utm_campaign=badge_info)

**CHANGE DESCRIPTION**
---
**Problem:**
PBM seems to require a 5-6 min delay before the first backup, otherwise restore to the latest oplog chunk fails. See [PBM-1265](https://perconadev.atlassian.net/issues/PBM-1265?jql=ORDER%20BY%20created%20DESC).

**Solution:**
Sleep for 6 minutes before running backups in pitr-physical test.

**CHECKLIST**
---
**Jira**
- [x] Is the Jira ticket created and referenced properly?
- [x] Does the Jira ticket have the proper statuses for documentation (`Needs Doc`) and QA (`Needs QA`)?
- [x] Does the Jira ticket link to the proper milestone (Fix Version field)?

**Tests**
- [x] Is an E2E test/test case added for the new feature/change?
- [x] Are unit tests added where appropriate?
- [x] Are OpenShift compare files changed for E2E tests (`compare/*-oc.yml`)?

**Config/Logging/Testability**
- [x] Are all needed new/changed options added to default YAML files?
- [x] Did we add proper logging messages for operator actions?
- [x] Did we ensure compatibility with the previous version or cluster upgrade process?
- [x] Does the change support oldest and newest supported MongoDB version?
- [x] Does the change support oldest and newest supported Kubernetes version?

[PBM-1265]: https://perconadev.atlassian.net/browse/PBM-1265?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: e2e-tests/pitr-physical/conf/some-name-rs0.yml [hunk 1]
```diff
@@ -54,11 +54,9 @@ spec:
       antiAffinityTopologyKey: none
     resources:
       limits:
-        cpu: 500m
-        memory: 1G
+        memory: 4G
       requests:
-        cpu: 100m
-        memory: 0.1G
+        memory: 1G
```
Comment (egegunes) [inline e2e-tests/pitr-physical/conf/some-name-rs0.yml:59]: @hors @tplavcic I'm not sure if we should change these but Dmytro pointed out that our previous memory limits are simply not enough to run physical restore.

**Comment by JNKPercona** [comment]
| Test name | Status |
| ------------- | ------------- |
| arbiter | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1494/4ee905ba/arbiter.log) |
| balancer | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1494/4ee905ba/balancer.log) |
| custom-replset-name | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1494/4ee905ba/custom-replset-name.log) |
| cross-site-sharded | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1494/4ee905ba/cross-site-sharded.log) |
| data-at-rest-encryption | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1494/4ee905ba/data-at-rest-encryption.log) |
| data-sharded | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1494/4ee905ba/data-sharded.log) |
| demand-backup | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1494/4ee905ba/demand-backup.log) |
| demand-backup-eks-credentials | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1494/4ee905ba/demand-backup-eks-credentials.log) |
| demand-backup-physical | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1494/4ee905ba/demand-backup-physical.log) |
| demand-backup-physical-sharded | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1494/4ee905ba/demand-backup-physical-sharded.log) |
| demand-backup-sharded | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1494/4ee905ba/demand-backup-sharded.log) |
| expose-sharded | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1494/4ee905ba/expose-sharded.log) |
| ignore-labels-annotations | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1494/4ee905ba/ignore-labels-annotations.log) |
| init-deploy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1494/4ee905ba/init-deploy.log) |
| finalizer | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1494/4ee905ba/finalizer.log) |
| ldap | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1494/4ee905ba/ldap.log) |
| ldap-tls | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1494/4ee905ba/ldap-tls.log) |
| limits | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1494/4ee905ba/limits.log) |
| liveness | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1494/4ee905ba/liveness.log) |
| mongod-major-upgrade | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1494/4ee905ba/mongod-major-upgrade.log) |
| mongod-major-upgrade-sharded | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1494/4ee905ba/mongod-major-upgrade-sharded.log) |
| monitoring-2-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1494/4ee905ba/monitoring-2-0.log) |
| multi-cluster-service | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1494/4ee905ba/multi-cluster-service.log) |
| non-voting | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1494/4ee905ba/non-voting.log) |
| one-pod | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1494/4ee905ba/one-pod.log) |
| operator-self-healing-chaos | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1494/4ee905ba/operator-self-healing-chaos.log) |
| pitr | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1494/4ee905ba/pitr.log) |
| pitr-sharded | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1494/4ee905ba/pitr-sharded.log) |
| pitr-physical | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1494/4ee905ba/pitr-physical.log) |
| recover-no-primary | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1494/4ee905ba/recover-no-primary.log) |
| rs-shard-migration | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1494/4ee905ba/rs-shard-migration.log) |
| scaling | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1494/4ee905ba/scaling.log) |
| scheduled-backup | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1494/4ee905ba/scheduled-backup.log) |
| security-context | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1494/4ee905ba/security-context.log) |
| self-healing-chaos | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1494/4ee905ba/self-healing-chaos.log) |
| service-per-pod | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1494/4ee905ba/service-per-pod.log) |
| serviceless-external-nodes | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1494/4ee905ba/serviceless-external-nodes.log) |
| smart-update | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1494/4ee905ba/smart-update.log) |
| split-horizon | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1494/4ee905ba/split-horizon.log) |
| storage | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1494/4ee905ba/storage.log) |
| tls-issue-cert-manager | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1494/4ee905ba/tls-issue-cert-manager.log) |
| upgrade | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1494/4ee905ba/upgrade.log) |
| upgrade-consistency | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1494/4ee905ba/upgrade-consistency.log) |
| upgrade-consistency-sharded-tls | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1494/4ee905ba/upgrade-consistency-sharded-tls.log) |
| upgrade-sharded | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1494/4ee905ba/upgrade-sharded.log) |
| users | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1494/4ee905ba/users.log) |
| version-service | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-psmdb-operator/PR-1494/4ee905ba/version-service.log) |
| We run 47 out of 47|

commit: https://github.com/percona/percona-server-mongodb-operator/pull/1494/commits/4ee905ba0d7ae5ee13a3e561d95dc8562688c733
image: `perconalab/percona-server-mongodb-operator:PR-1494-4ee905ba`

**Review by hors — APPROVED** [review]

File: e2e-tests/pitr-physical/run [hunk 2]
```diff
@@ -172,7 +174,6 @@ main() {
 
 	check_recovery $backup_name_minio-1 date "${last_chunk}" "-2nd" "$cluster"
 
-	run_backup $backup_name_minio 2 logical
```
Comment (ptankov) [inline e2e-tests/pitr-physical/run:175]: why no need for logical anymore?

Comment (egegunes) [reply e2e-tests/pitr-physical/run:175]: it was there as a workaround for PBM bug that doesn't start PiTR routine until a logical backup is created even if there's a physical backup. now the bug is fixed and running a logical backup is not necessary

**Review by ptankov — APPROVED** [review]

**Review by hors — APPROVED** [review]

**Pull request merged**